kernel/common/get_slice.rs

```diff
@@ -1,8 +1,6 @@
-use core::slice;
-use core::str;
 use collections::range::RangeArgument;
 use core::ops::Range;
-use core::cmp::{max, min};
+use core::cmp::min;
```

Comment:
it's also in a new `slice` module which I made, originally to hold both `OptionSlice` and `GetSlice` at the time, but instead it holds my `AsOption` trait which is what powered up `GetSlice` to accept both `usize` and `Option<usize>` parameters generically. So, I'm concerned of losing my work.